angularjs-portal-frame/src/main/webapp/css/buckyless/portlet-frame.less

```diff
@@ -0,0 +1,50 @@
+/* Portlet Frame */
+
+/*  
+The portlet frame is the div that houses the main content of the page. 
+It contains a header div (.portlet-header), usually with a title, description, and background image. 
+It also contains a body div (.portlet-body), that houses the content.
+*/
+
+.portlet-frame {
```

Comment:
That's a decent idea, but we already have a my-uw namespace (that .portlet-frame is under) and i don't think we should have too many layers of namespaces. The idea for this file is just to outline the portlet window, i.e. this basic structure:
![image](https://cloud.githubusercontent.com/assets/1919535/6136676/467ecf50-b138-11e4-9569-2878595343fe.png)
(this example is coming to the toolkit very soon, pull request is up)

For this reason, I don't think this file should get a ton added to it in the future, so might not need namespacing.